src/merge.rs

```diff
@@ -12,6 +12,12 @@ pub enum MergeValue {
         zero_bits: H256,
         zero_count: u8,
     },
+    #[cfg(feature = "trie")]
+    ShortCut {
+        key: H256,
+        value: H256,
+        height: u8,
+    },
```

Comment:
How about add some comment to this enum variant, such as something like:
> A shortcut is the highest subtree root that contains only one key should store the key-value pair instead of its children (the shortcut is still the value of the subtree root that contains 1 non-default value at height 0). With this storage rule, we don’t have to store branches in a sparse tree which significantly reduces the quantity of data to be stored.